### DISCUSSION TITLE
Add 4.x.x+ to maven repository

### DISCUSSION THREAD

**Discussion by Y2Kwastaken** [nitrite/nitrite-java#813] [General]
I noticed that there wasn't a ton of documentation for 4.x.x+ in all of the java docs and provided by the maven repo. https://repo.maven.apache.org/maven2/org/dizitart/nitrite/ version support ends at 3.4.4

The releases on github also haven't been bumped at all

**Comment by anidotnet** [comment]
Version 4.x is still under development. Once it is finalized, I'll release it in maven along with documentation.
